File: c++/1015.cpp
#include<stdio.h>
#include<math.h>
int main ()
 {
     double A,B,C,D,E;
     scanf("%lf%lf%lf%lf",&A,&B,&C,&D);
   E=sqrt(((C-A)*(C-A))+(D-B)*(D-B));
   printf("%.4f\n",E);
    return 0;
}

File: 1015.py
import math
a,b=input().split(" ")
a=float(a)
b=float(b)
c,d=input().split(" ")
c=float(c)
d=float(d)
print("{:.4f}".format(math.sqrt((c-a)*(c-a)+(d-b)*(d-b))))
